packages/renderer/src/lib/kube/KubernetesDashboard.svelte

```diff
@@ -0,0 +1,143 @@
+import KubernetesDashboardResourceCard from './KubernetesDashboardResourceCard.svelte';
+import KubernetesEmptyPage from './KubernetesEmptyPage.svelte';
+
+let welcome = $derived($kubernetesCurrentContextState.error === NO_CURRENT_CONTEXT_ERROR);
+let currentContextName = $derived($kubernetesContexts.find(context => context.currentContext)?.name);
+
+let nodeCount = $state<number>(0);
+let deploymentCount = $state<number>(0);
+let serviceCount = $state<number>(0);
+let ingressCount = $state<number>(0);
+let routeCount = $state<number>(0);
+let pvcCount = $state<number>(0);
+let configMapCount = $state<number>(0);
+let secretCount = $state<number>(0);
+
+let unsubscribers: Unsubscriber[] = [];
+
+onMount(async () => {
+  unsubscribers.push(
+    kubernetesCurrentContextNodes.subscribe(nodes => {
+      nodeCount = nodes.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextDeployments.subscribe(deployments => {
+      deploymentCount = deployments.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextServices.subscribe(services => {
+      serviceCount = services.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextIngresses.subscribe(ingresses => {
+      ingressCount = ingresses.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextRoutes.subscribe(routes => {
+      routeCount = routes.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextPersistentVolumeClaims.subscribe(pvcs => {
+      pvcCount = pvcs.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextConfigMaps.subscribe(configMaps => {
+      configMapCount = configMaps.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextSecrets.subscribe(secrets => {
+      secretCount = secrets.length;
+    }),
+  );
+});
+
+onDestroy(async () => {
+  unsubscribers.forEach(unsubscriber => unsubscriber());
+});
+</script>
+
+<NavPage searchEnabled={false}  title="About Kubernetes">
+  <svelte:fragment slot="additional-actions">
+    {#if !welcome}
+    <KubeApplyYamlButton />
+    {/if}
+  </svelte:fragment>
+
+  <svelte:fragment slot="bottom-additional-actions">
+    <div class="flex grow justify-end">
+      <KubernetesCurrentContextConnectionBadge />
+    </div>
+  </svelte:fragment>
+
+  <div class="flex min-w-full h-full" slot="content">
+    {#if welcome}
+        <KubernetesEmptyPage />
+    {:else}
+    <div class="flex flex-col space-y-4 min-w-full overflow-y-auto">
+        <div class="flex flex-col gap-4 p-5">
+            <span>Here you can manage and interact with Kubernetes clusters with features like connect to clusters,
+            view workloads like deployments and services.</span>
+
+        <span>Get up and running by clicking one of the menu items!</span>
+
+        <Link class="place-self-start" on:click={() => window.openExternal('https://podman-desktop.io/docs/kubernetes')}>Kubernetes documentation</Link>
```

Comment:
issue: use a function for the click (like openKubernetesDocumentation)

why ? b/c there is a linter rule saying you need to provide the returning type and it's not possible inlining

Reply:
Fixed (in doc link, resource card, and guide cards.